dvc/machine/__init__.py

```diff
@@ -25,6 +25,17 @@
 
 RESERVED_NAMES = {"local", "localhost"}
 
+DEFAULT_STARTUP_SCRIPT = """#!/bin/bash
+sudo apt-get update
+sudo apt-get install --yes python3 python3-pip
+python3 -m pip install --upgrade pip
+
+cd /etc/apt/sources.list.d
+sudo wget https://dvc.org/deb/dvc.list
+sudo apt-get update
+sudo apt-get install --yes dvc
```

Comment:
Perhaps use `pushd`/`popd` to ensure the cwd is not changed?

Reply:
Terraform runs the startup script in its own shell instance, so I don't think changing the cwd will actually affect anything else. But it's probably still good to make this change anyways in case we add more actions to the install script later